### PR TITLE
Override 'type' for close button

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Close button should be a regular button and not "submit" which is the default.

## Why?

Avoid closing the form "on enter".

**Ticket(s): []**

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1219** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1219" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1219?utm_source=stack-comment).

**Review by alebedev — APPROVED** [review]

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1219?utm_source=stack-comment).


Job Reference: Nh8eAqZZPFbUDuMar0xQ

**Pull request merged**
